rest/specs/src/main/filtered-resources/openapi/openapi.yaml

```diff
@@ -172,6 +182,11 @@ components:
       maxLength: 80
       pattern: '^([1-9]([0-9]+)?)\.(0|([1-9]([0-9]+)?))(\.(0|([1-9]([0-9]+)?))(-.*)?)?$'
       example: 1.2, 2.4.3, 0.2.5-SNAPSHOT
+    Date-Time:
+      description: the date-time notation as defined by RFC 3339, section 5.6
+      type: string
+      format: date-time
+      example: '2017-07-21T17:32:28Z'
```

Comment:
✏️ Don't need the quotes

Reply:
You do need the quotes because swagger can't handle the `:`s otherwise

Reply:
Wouldn't the example for the timestamp in the ErrorMessage also be an issue then?

Reply:
I tested it myself and didn't have any issues in Swagger after removing the quotes.

Reply:
After looking into this further, the quotes are not needed if the date time contains the time offset after the T, so both ways are valid...